### ISSUE TITLE
No info after update to 1.2.1

### ISSUE THREAD

**Issue by smaragdschlange** [iobroker-community-adapters/ioBroker.info#18]
Thanks for the adapter. It makes things a little easier :-) and it used to work like a charm up to version 1.1.3.
Unfortunately after I've updated to version 1.2.1 there's no information shown on the info page anymore (even the clock shows only 12:00).
I've tried clearing the browser's cache, reloading the ioBroker webui, deleting and recreating the instance of the adapter and even restarting the ioBroker server. Starting the instance in debug mode shows only the following entries:

info.0 | 2019-03-02 10:57:23.324 | info | starting. Version 1.2.1 in /opt/iobroker/node_modules/iobroker.info, node: v8.15.0
-- | -- | -- | --
info.0 | 2019-03-02 10:57:23.282 | debug | statesDB connected
info.0 | 2019-03-02 10:57:23.259 | debug | objectDB connected

I am running two ioBroker servers: One on a NUC with Debian 8, Node.js: v8.15.0 and NPM: v6.4.1, another one on a Tinkerboard S with TinkerOs, Node.js: v8.12.0 and NPM: 6.4.1.
I also tried installing version 1.2.0 without any luck. Setting the adapter back to version 1.1.3 didn't fix the issue either. 
Is there anything I could have missed when updating to the latest version? Or could there be an issue with another adapter that has been updated recently (e.g. Script Engine)?

**Comment by ldittmar81** [comment]
It sounds like a Javascript error on browser. Can you open the browser console "F12" and tell me if there is an error?

**Comment by smaragdschlange** [comment]
There seems to be an error. This is written in the console when I open the info page:
Unsibscribe: system.host.* admin.js:1387
Object { topic: "mg-loadFrame", data: {…} }
adminConfig.js:17
Unknown event: [object Object] adminConfig.js:24
Quellübergreifende (Cross-Origin) Anfrage blockiert: Die Gleiche-Quelle-Regel verbietet das Lesen der externen Ressource auf https://nodejs.org/dist/index.json. (Grund: CORS-Anschlag schlug fehl).[Weitere Informationen]
TypeError: NetworkError when attempting to fetch resource.

**Comment by ldittmar81** [comment]
Wich Browser? Firefox?

**Comment by smaragdschlange** [comment]
Yes, Firefox 65.0.2 (64-Bit)...
...and yes, it works with Chrome Version 72.0.3626.119 (Offizieller Build) (64-Bit).

Oh, stupid me!
I found out what caused the issue. I am using NoScript-plugin on Firefox. It did not show me that anything was blocked but it turned out that somehow it decided to block node.js after I updated the info adapter! I just set node.js to trusted and now the info page workes great again.

Thanks for your help and sorry for bothering you.
